src/core/src/energy/global/ewald.rs

```diff
@@ -704,6 +704,10 @@ impl Ewald {
 }
 
 impl GlobalPotential for Ewald {
+    fn get_cutoff(&self) -> Option<f64> {
```

Comment:
Sure. I am using the same name here that we have for the pair potential. Might change it as well.